.github/workflows/test.yaml

```diff
@@ -1,11 +1,11 @@
-name: Test
+name: Test - NNBD branch
 
 on:
   # Run CI on pushes to the main branch, and on PRs against main.
   push:
-    branches: [ master ]
+    branches: [ nnbd ]
```

Comment:
@srawlins – should this be `master`? Oops?

Reply:
oops; working on this